dashboard.go

```diff
@@ -28,6 +28,7 @@ type Dashboard struct {
 	Model     map[string]interface{} `json:"dashboard"`
 	Folder    int64                  `json:"folderId"`
 	Overwrite bool                   `json:"overwrite"`
+	Message   string                 `json:"message"`
```

Comment:
Since it's an attribute that's passed in the POST but not really part of a dashboard. I'd add a comment to say that this is only used in a save operation and that it will always be empty when fetching a dashboard